component/azstorage/azstorage.go

```diff
@@ -326,15 +326,15 @@ func (az *AzStorage) ReadFile(options internal.ReadFileOptions) (data []byte, er
 	return az.storage.ReadBuffer(options.Handle.Path, 0, 0)
 }
 
-func (az *AzStorage) ReadInBuffer(options internal.ReadInBufferOptions) (len int, err error) {
+func (az *AzStorage) ReadInBuffer(options internal.ReadInBufferOptions) (length int, err error) {
```

Comment:
So root-cause is libfuse always converting the C buffer to 2^30 cap slice. We shall restrict that to create rather a smaller slice just in case anywhere we try to use that inside sdk also it will result into buffer overrun. 
we can convert that (1 << 30)  to (1 << 22) ~= 4MB size. As of now we know libfuse always reads in blocks of 128K only so ideal size shall be (1 << 16)